app/models/group.rb

```diff
@@ -17,31 +17,40 @@ def self.create_with_admin(params, user)
     group
   end
 
-  def admins
-    User.in_group(self).as(:admin)
-  end
-
   def admin?(user)
     user.in_group?(self, as: 'admin')
   end
 
-  def member?(user)
+  def full_member?(user)
     user.in_group?(self) unless user.in_group?(self, as: 'pending')
   end
 
+  def member?(user)
+    user.in_group?(self, as: 'member')
+  end
+
   def make_admin(user)
+    users.delete(user)
```

Comment:
Do we have a way to remove admin permissions from a member?

Reply:
The there is currently no way to "unadmin" an admin in a group.

Reply:
Alright :)